test/cri-containerd/runpodsandbox_test.go

```diff
@@ -996,6 +999,29 @@ func createExt4VHD(ctx context.Context, t *testing.T, path string) {
 	}
 }
 
+func createNTFSVHD(vhdPath string, sizeGB uint32, t *testing.T) error {
+	if err := vhd.CreateVhdx(vhdPath, sizeGB, 1); err != nil {
+		return errors.Wrap(err, "failed to create VHD")
+	}
+
+	vhd, err := vhd.OpenVirtualDisk(vhdPath, vhd.VirtualDiskAccessNone, vhd.OpenVirtualDiskFlagNone)
+	if err != nil {
+		return errors.Wrap(err, "failed to open VHD")
+	}
+	defer func() {
+		err2 := windows.CloseHandle(windows.Handle(vhd))
+		if err == nil {
```

Comment:
This conditional won't work properly unless you name the function's return argument `err`.

Reply:
Fixed